test/studies/ml/lib/MNISTTester.chpl

```diff
@@ -0,0 +1,158 @@
+import Chai as torch;
+import Tensor as tn;
+use Tensor;
+import Math;
+import MNIST;
+import IO;
+import Time;
+
+config param perfTest = false;
+
+tn.seedRandom(0);
+
+config const dataPath = "./data";
+
+proc forward(ref net,x: Tensor(?), lb: int) {
+    const output = net.forwardProp(x);
+    const loss = -Math.log(output[lb]);
+    const acc = tn.argmax(output.data) == lb;
+    return (output,loss,acc);
+}
+
+proc train(ref net, data: [] (Tensor(3),int), lr: real = 0.005) {
```

Comment:
Is this proc used, or just the other `train` below? If not, I'd recommend removing it.

Reply:
This proc is used as a helper function for the one below it.